### PR TITLE
subsystem-bench: polish imports

### PR DESCRIPTION
**Pull request by AndreiEres**

### CODE REVIEW

**Review by sandreim — APPROVED** [review]

File: polkadot/node/subsystem-bench/src/subsystem-bench.rs [hunk 1]
```diff
@@ -287,7 +210,6 @@ fn main() -> eyre::Result<()> {
 		.filter(Some("polkadot_overseer"), log::LevelFilter::Error)
 		.filter(None, log::LevelFilter::Info)
 		.format_timestamp_millis()
-		// .filter(None, log::LevelFilter::Trace)
```
Comment (alindima) [inline polkadot/node/subsystem-bench/src/subsystem-bench.rs:290]: I think this was left intentionally for easier debugging

File: polkadot/node/subsystem-bench/src/subsystem-bench.rs [hunk 2]
```diff
@@ -54,16 +54,41 @@ fn le_5000(s: &str) -> Result<usize, String> {
...
+#[derive(Debug, Clone, Parser, Serialize, Deserialize)]
+#[command(rename_all = "kebab-case")]
+pub enum TestObjective {
+	/// Benchmark availability recovery strategies.
+	DataAvailabilityRead(DataAvailabilityReadOptions),
+	/// Benchmark availability and bitfield distribution.
+	DataAvailabilityWrite,
+	/// Benchmark the approval-voting and approval-distribution subsystems.
+	ApprovalVoting(ApprovalsOptions),
+	Unimplemented,
```
Comment (alindima) [inline polkadot/node/subsystem-bench/src/subsystem-bench.rs:64]: why do we have unimplemented?

**Review by alindima — APPROVED** [review]
Looks good! The PR title is quite misleading though :D

File: polkadot/node/subsystem-bench/src/subsystem-bench.rs (hunk 1 above)
Comment (AndreiEres) [reply polkadot/node/subsystem-bench/src/subsystem-bench.rs:216]: It looks like a leftover :-)

File: polkadot/node/subsystem-bench/src/subsystem-bench.rs (hunk 2 above)
Comment (AndreiEres) [reply polkadot/node/subsystem-bench/src/subsystem-bench.rs]: Can't answer the question. It's not a part of my PR after I rebased the branch. And It was not me who implemented it before.

**Pull request merged**